.github/workflows/ruby.yml

```diff
@@ -6,7 +6,6 @@ jobs:
     strategy:
       matrix:
         ruby:
-          - '2.4'
```

Comment:
ActiveSupport >=6 means no EoL'd rubies. Which, honestly, is fine since this should probably end up being a whole new major release anyhow. `1.0.0.beta.1`, maybe? 😄